mne_bids/read.py

```diff
@@ -73,6 +73,30 @@ def _read_raw(raw_fpath, electrode=None, hsp=None, hpi=None, config=None,
     return raw
 
 
+def _handle_info_reading(sidecar_fname, raw):
+    """Read associated sidecar.json and populate raw.
+
+    Handle PowerLineFrequency of recording.
+    """
+    with open(sidecar_fname, "r") as f:
+        sidecar_json = json.load(f)
+
+    line_freq = sidecar_json["PowerLineFrequency"]
+    if line_freq is not None:
+        if line_freq != raw.info["line_freq"]:
+            raise ValueError("Line frequency in sidecar json does "
+                             "not match the info datastructure of "
+                             "the mne.Raw.")
+        raw.info["line_freq"] = line_freq
```

Comment:
this line (90) is no needed, right? --> in line 86 you check what happens if `line_freq` and the raw info are NOT the same ... so everything afterwards means that they ARE the same. so no need to reassign them to be the same again.